build.xml

```diff
@@ -26,7 +26,7 @@
   <property name="ts.home" value="${basedir}"/>
   <property name="deliverable.name" value="debugging-tck"/>
   <property name="eclipse.deliverable.name" value="eclipse-debugging-tck"/>
-  <property name="deliverable.version" value="1.0.0"/>
+  <property name="deliverable.version" value="2.0.0"/>
```

Comment:
Changes looks fine, probably we can update the copyright year from "Copyright (c) 2018 Oracle" to "Copyright (c) 2018, 2020 Oracle"

Reply:
good catch, done!